build.sbt

```diff
@@ -21,7 +21,7 @@ libraryDependencies ++= Seq(
   "edu.arizona.sista" % "processors" % "3.0" classifier "models",
   "org.slf4j" % "slf4j-api" % "1.7.2",
   "org.slf4j" % "slf4j-log4j12" % "1.7.2",
-  "org.scalatest" %% "scalatest" % "1.9.1" % "test",
+  "org.scalatest" %% "scalatest" % "2.2.6" % "test",
```

Comment:
Hmm.. it does seem to break the tests - anything that runs with a SparkContext is affected by this - we end up with a bunch of errors like this:
```